WordPressAuthenticator.podspec

```diff
@@ -32,7 +41,7 @@ Pod::Spec.new do |s|
   s.dependency 'SVProgressHUD', '2.2.5'
 
   s.dependency 'Gridicons', '~> 0.15'
-  s.dependency 'GoogleSignInRepacked', '4.1.2'
+  s.dependency 'GoogleSignIn', '4.1.2'
```

Comment:
We can probably update this too since this is an old version but better to do one thing at a time 😄